components/Button/Button.js

```diff
@@ -0,0 +1,12 @@
+import * as styles from "./Button.module.scss";
+
+export default function Button({ children, onClick }) {
+  return (
+    <button
+      className={styles.button}
+      onClick={onClick}
```

Comment:
you know whats happened here? I'm asking cause I got some difficult to understand the **onClick** working on this case